src/plugins/data/public/search/errors/timeout_error.tsx

```diff
@@ -78,9 +78,7 @@ export class SearchTimeoutError extends KbnError {
   private onClick(application: ApplicationStart) {
     switch (this.mode) {
       case TimeoutErrorMode.UPGRADE:
-        application.navigateToApp('management', {
-          path: `/kibana/indexPatterns`,
-        });
+        application.navigateToUrl('https://www.elastic.co/subscriptions');
```

Comment:
This issue states that the redirect should go elsewhere on cloud. Is it possible?

https://github.com/elastic/kibana/issues/70376

Reply:
From the linked issue:

> I don't consider it a priority issue but logging it for tracking.

I think it's on the radar that having a generalized service for this is getting more and more important, but I don't feel like it's our realm of responsibility to provide this in this issue.